example/views.py

```diff
@@ -185,7 +187,16 @@ class NoFiltersetEntryViewSet(EntryViewSet):
 
 class AuthorViewSet(ModelViewSet):
     queryset = Author.objects.all()
-    serializer_class = AuthorSerializer
+    serializer_classes = {
```

Comment:
Is this new `serializer_classes` functionality something that perhaps belongs in upstream DRF?